policyengine_us/variables/gov/states/ar/tax/income/ar_taxable_capital_gains.py

```diff
@@ -10,17 +10,29 @@ class ar_taxable_capital_gains(Variable):
     defined_for = StateCode.AR
 
     def formula(person, period, parameters):
+        # Line 1-3 - long term capital gain or loss
         lt_capital_gains = person("long_term_capital_gains", period)
-        net_st_capital_gains = person("short_term_capital_gains", period)
-        # Can be capital loss
-        net_cap_gain = lt_capital_gains - net_st_capital_gains
+        # Line 4-6 - short term capital loss
+        st_capital_gains = person("short_term_capital_gains", period)
+        capital_loss = -st_capital_gains
```

Comment:
then you can subtract directly. i think it's sufficiently conventional to consider losses only non-negative values that we can use this directly.
```suggestion
        st_capital_loss = max_(-st_capital_gains, 0)
```